osgp/shared/osgp-dto/src/main/java/org/opensmartgridplatform/dto/da/measurements/MeasurementDto.java

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright 2019 Smart Society Services B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.opensmartgridplatform.dto.da.measurements;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public class MeasurementDto implements Serializable {
+
+    private static final long serialVersionUID = -7800836859182549458L;
+
+    private final List<MeasurementElementDto> measurementElements;
+
+    public MeasurementDto(final List<MeasurementElementDto> measurementElements) {
+        this.measurementElements = new ArrayList<>(measurementElements);
+    }
+
+    public List<MeasurementElementDto> getMeasurementElements() {
+        return new ArrayList<>(this.measurementElements);
+    }
+
+    @Override
+    public boolean equals(final Object obj) {
```

Comment:
Many DTO and value object classes have methods for equals and hashCode. Also supply a toString method for those classes.